README.md

```diff
@@ -1,25 +1,25 @@
-# ssp-client
+# rsp-client
 [![Build Status](https://travis-ci.org/redhat-developer/rsp-client.svg?branch=master)](https://travis-ci.org/redhat-developer/rsp-client)
-A simple client for the Simple Server Protocol (SSP) written in typescript.
+A simple client for the Runtime Server Protocol (RSP) written in typescript.
 
 Implements the API described [here](https://github.com/robstryker/org.jboss.tools.ssp/blob/master/api/src/main/java/org/jboss/tools/ssp/api/SSPClient.java) and [here](https://github.com/robstryker/org.jboss.tools.ssp/blob/master/api/src/main/java/org/jboss/tools/ssp/api/SSPServer.java).
 
-See the [SSP repo](https://github.com/robstryker/org.jboss.tools.ssp) for further details about SSP.
+See the [RSP repo](https://github.com/robstryker/org.jboss.tools.ssp) for further details about RSP.
```

Comment:
The url for the repo is also changed.
Correct URL: https://github.com/redhat-developer/rsp-server